README.md

```diff
@@ -7,7 +7,45 @@
 [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/MotionAnimator.svg)](https://cocoapods.org/pods/MotionAnimator)
 [![Platform](https://img.shields.io/cocoapods/p/MotionAnimator.svg)](http://cocoadocs.org/docsets/MotionAnimator)
 
-
+<table>
+  <tr><td>🎉</td><td>Implicit and explicit additive animations.</td></tr>
+  <tr><td>🎉</td><td>Parameterized motion with the <a href="https://github.com/material-motion/motion-interchange-objc">Interchange</a>.</td></tr>
+  <tr><td>🎉</td><td>Provide velocity to animations directly from gesture recognizers.</td></tr>
+  <tr><td>🎉</td><td>Maximize frame rates by relying more on Core Animation.</td></tr>
+  <tr><td>🎉</td><td>Animatable properties are Swift enum types.</td></tr>
+  <tr><td>🎉</td><td>Consistent model layer value expectations.</td></tr>
+</table>
+
+The following properties can be implicitly animated using the MotionAnimator on iOS 8 and up:
+
+<table>
+  <tr><td>☑️</td><td>CALayer <tt>anchorPoint</tt></td></tr>
+  <tr><td>☑️</td><td>CALayer <tt>backgroundColor</tt></td><td>☑️</td><td>UIView <tt>backgroundColor</tt></td></tr>
+  <tr><td>☑️</td><td>CALayer <tt>bounds</tt></td><td>☑️</td><td>UIView <tt>bounds</tt></td></tr>
```

Comment:
Nit: Consider removing the checkboxes? I don't think they provide any new information since the table is explicitly a listing of supported properties.  For VoiceOver users it's also a little distracting, this reads as "check, checkbox", then "CALayer anchorpoint", then "check, checkbox", then "CALayer backgroundColor".

Reply:
Done.